types/react-datepicker/index.d.ts

```diff
@@ -127,5 +127,12 @@ export interface ReactDatePickerProps {
 	withPortal?: boolean;
 	yearDropdownItemNumber?: number;
 }
-declare const ReactDatePicker: React.ClassicComponentClass<ReactDatePickerProps>;
+
+declare class ReactDatePicker extends React.Component<ReactDatePickerProps> {
+    readonly setBlur: () => void;
+    readonly setFocus: () => void;
+    readonly setOpen: (open: boolean) => void;
```

Comment:
Looks like `setOpen` is missing the optional parameter `skipSetBlur`

Reply:
thanks, I have added it now